spec/models/organization_spec.rb

```diff
@@ -24,12 +24,14 @@
 #  created_at               :datetime         not null
 #  updated_at               :datetime         not null
 #  account_request_id       :integer
+#  ndbn_member_id           :integer
 #
 
 RSpec.describe Organization, type: :model do
   let(:organization) { create(:organization) }
   context "Associations >" do
     it { should have_many(:item_categories) }
+    it { should belong_to(:ndbn_member).with_foreign_key(:ndbn_member_id).optional }
```

Comment:
I think we need the classname in here too -- `it { should belong_to(:ndbn_member).class_name("NDBNMember").with_foreign_key(:ndbn_member_id).optional }`